mat64/symmetric_test.go

```diff
@@ -119,6 +123,19 @@ func (s *S) TestSymAdd(c *check.C) {
 			}
 		}
 	}
+
+	type addSymer interface {
```

Comment:
I was thinking in the minimal scope (in the definition of `method`), but this is fine.